src/libraries/System.IO.FileSystem/tests/Directory/GetFileSystemEntries_str_str.cs

```diff
@@ -703,19 +703,19 @@ public void SearchPatternCaseSensitive()
         {
             DirectoryInfo testDir = Directory.CreateDirectory(GetTestFilePath());
             string testBase = GetTestFileName();
-            testDir.CreateSubdirectory(testBase + "aBBb");
-            testDir.CreateSubdirectory(testBase + "aBBB");
+            testDir.CreateSubdirectory(testBase + "aBBBBBBBBBBb");
```

Comment:
Why do you need larger names?

Reply:
this will fail whenever the test file name contains two capital B's in a row. previously that didn't happen as GUID letters formatted as lower case. Initially I was using capital letters in the generated names so it failed approx every 200 runs. I changed to lower case for some reason but seemed reasonable to make it longer in case we change again.

Reply:
Sounds quite hard to figure out when an issue reporting the failing test gets filed. Consider adding a comment.